jannovar-core/src/main/java/de/charite/compbio/jannovar/impl/parse/refseq/RefSeqParser.java

```diff
@@ -227,7 +221,7 @@ private void loadFASTA(Map<String, TranscriptModelBuilder> builders, String path
 			while ((record = parser.next()) != null) {
 				numRecords++;
 				String transcriptId = record.getAttributes().get("transcript_id");
-				if (onlyCurated && (transcriptId == null || transcriptId.startsWith("X"))) {
+				if (onlyCurated && (transcriptId != null && transcriptId.startsWith("X"))) {
```

Comment:
can you remove the non-necessary brackets?

Reply:
done